lib/src/impl/rest/channels.dart

```diff
@@ -1,14 +1,16 @@
 import 'dart:async';
-
+import 'dart:collection';
+import 'package:pedantic/pedantic.dart';
 import 'package:ably_flutter_plugin/ably.dart';
 import 'package:ably_flutter_plugin/src/impl/rest/rest.dart';
 import 'package:ably_flutter_plugin/src/spec/spec.dart' as spec;
 import 'package:flutter/services.dart';
 
 import '../platform_object.dart';
 
-
-class RestPlatformChannel extends PlatformObject implements spec.RestChannel{
+class RestPlatformChannel extends PlatformObject implements spec.RestChannel {
+  static const String clientConfiguredAuthenticationProviderRequestFailed =
```

Comment:
@zoechi A new commit (e9e0939) has been cherry-picked from another latest branch into `feature/realtime-authcallback`.

You may rebase and use that constant

Reply:
`PlatformException` has `final String code`, but `ErrorCodes` has `static const int authCallbackFailure`
Does this type inconsistency `String`/`int` make any sense?

Reply:
This discussion is continued either https://github.com/ably/ably-flutter/pull/38#discussion_r484375487 or https://github.com/ably/ably-flutter/pull/38#discussion_r484379978